# Protocal prefix
string VEL = "V"
string POS = "P"
string ENC = "E"
string START = "I"
string STOP = "S"
string ZERO = "Z"
string LIMIT = "L"

std_msgs/Header header

uint8[] byte_msg